lib/consumer.js

```diff
@@ -112,7 +112,7 @@ Consumer.prototype.init = function () {
             return self.emit('error', err);
         }
 
-        if (self.options.fromOffset)
+        if (!self.options.fromOffset)
```

Comment:
@OndroNR I think I misinterpeted the fromOffset variable. (I assumed that setting the fromOffset to true that the offset for this client should be fetched from Kafka) Thank you for your feedback, I will close the pull request/